docs/config.rst

```diff
@@ -0,0 +1,122 @@
+Configuration Options
+=====================
+``schema.registry.url``
```

Comment:
Based on the fact that it looks like the configs and their defaults are scattered throughout the code base and seem to lack descriptions in the the source, I assume you built this manually based on the README? Are we ok only listing those options or should we do a more exhaustive listing? If the more exhaustive version, do we know if the sample properties file covers all the configs?